src/analytics/redux/tracked_events.js

```diff
@@ -89,9 +72,7 @@ function setIsDraggingBlock( isDragging ) {
  * @return {void}
  */
 function trackBlockMoved( clientIds, action ) {
-	const block = select( 'core/block-editor' ).getBlocksByClientId(
-		clientIds?.[ 0 ]
-	)?.[ 0 ];
+	const block = select( 'core/block-editor' ).getBlock( clientIds?.[ 0 ] );
```

Comment:
If we expect that in some cases the `clientIds` is not defined (in relation to the statement `clientIds?.[ 0 ]`), probably we should check if the block returned is defined and prevent sending the event, wdyt?

Reply:
Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4833/commits/64337fa237e45ecd20ad4c16bb2020876f3f5dbb